<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStrategy</class>
 <widget class="QWidget" name="NewStrategy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>326</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>326</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建策略</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 245);</string>
  </property>
  <widget class="QLabel" name="label_user_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>期货账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_strategy_id">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>策略编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_a_instrument">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A合约代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_instrument">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B合约代码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_user_id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a_instrument">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_b_instrument">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>240</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 100 10pt &quot;微软雅黑&quot;;
	background-color: rgb(255, 255, 255);
	border: 1px solid #6D6D6D;
	color: rgb(0, 0, 0);
}

QPushButton::pressed{
	background-color: rgb(109, 109, 109);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 100 10pt &quot;微软雅黑&quot;;
	background-color: rgb(255, 255, 255);
	border: 1px solid #6D6D6D;
	color: rgb(0, 0, 0);
}

QPushButton::pressed{
	background-color: rgb(109, 109, 109);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_msg">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>192</y>
     <width>237</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_strategy_id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_user_id</tabstop>
  <tabstop>lineEdit_strategy_id</tabstop>
  <tabstop>lineEdit_a_instrument</tabstop>
  <tabstop>lineEdit_b_instrument</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
